.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+1. **What type of issue would you like to report**?
+
+- [ ] Bug
```

Comment:
It was initially added as `[ ] Xyz`, how about utilizing actual Github-flavored checkboxes? I accidentally have committed change in this file, while I just wanted to preview it. Fell free to revert.